docs/operating-systems/course-tools.md

```diff
@@ -201,70 +197,94 @@ You want your output to say you have the clang version.
           gcc -c hello.c
           
       clean:
-          rm hello.o hello.exe```
-9. Navigate to the directory in your terminal, enter make and see the program run!
+          rm hello.o hello.exe
+      ```
+ 
+Navigate to the directory in your terminal, and run the make command.
 
 ### Docker
-- Navigate to https://docs.docker.com/desktop/install/windows-install/, get the installer, and follow the instructions.
+Navigate to https://docs.docker.com/desktop/install/windows-install/ to install Docker.
```

Comment:
I would recommend you change this link to the format you used above in the different languages.